CHANGES.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG
 =========
 
+## 5.4.0 (2018-4-26)
+* @RobGraham Added `distribution()` support for DataDog v6
```

Comment:
I believe you may be the first person ever to add to the changelog themselves- thanks.  :)